lib/internal/main/embedding.js

```diff
@@ -3,15 +3,15 @@ const {
   prepareMainThreadExecution,
   markBootstrapComplete,
 } = require('internal/process/pre_execution');
-const { isSea } = internalBinding('sea');
+const { isExperimentalSeaWarningDisabled, isSea } = internalBinding('sea');
```

Comment:
I think we can have just something like `isExperimentalSeaWarningNeeded()` and there is no need for `isSea()` now anyway?